src/Fields/Fields.jl

```diff
@@ -56,6 +56,9 @@ const CenterExtrudedFiniteDifferenceField{V, S} = Field{
     S,
 } where {V <: AbstractData, S <: Spaces.CenterExtrudedFiniteDifferenceSpace}
 
+const ColumnField =
+    Field{V, S} where {V <: DataLayouts.VF, S <: Spaces.FiniteDifferenceSpace}
```

Comment:
```suggestion
const ColumnField =
    Field{V, S} where {V <: DataLayouts.DataColumn, S <: Spaces.FiniteDifferenceSpace}
```
or maybe we could just rename `FiniteDifferenceField` above?

Reply:
I'm fine with either one. But aren't we suppose to restrict this to `DataColumn`? `FiniteDifferenceField ` is only restricted to `AbstractData`.

Reply:
I made the suggested change to `DataColumn`.

Reply:
we only support finite difference fields as columns at the moment

Reply:
I'm just not keen on having two aliases for the same thing

Reply:
Actually, lets rename the above one to `ColumnFiniteDifferenceField`

Reply:
Done.